lib/reducers/ui.js

```diff
@@ -195,7 +196,7 @@ const reducer = (state = initial, action) => {
               ret.bell = config.bell;
             }
 
-            if (config.bellSoundURL) {
+            if (config.bellSoundURL !== state.bellSoundURL) {
```

Comment:
I think this is not necessary to compare with value in the state.

Reply:
This one does have a specific reason as i initially thought the same before I implemented this. Without comparing against the current state, removing the `bellSoundURL` property from the config file would not properly restore the sound back to a default sound. Instead of only updating the bellSoundURL field if the value exists, we check if it has changed and updated accordingly.